chia/protocols/timelord_protocol.py

```diff
@@ -74,7 +74,7 @@ class NewEndOfSubSlotVDF(Streamable):
 
 @dataclass(frozen=True)
 @streamable
-class RequestCompactProofOfTime:
+class RequestCompactProofOfTime(Streamable):
```

Comment:
I would expect the annotation could add the base class too, no? (maybe for later)

Reply:
The annotation could do the dataclass wrapping as well, but the comment in `def streamable` states that this is intended for linters so they could recognize the methods of the base class as well as the `__init__` method provided by `dataclass`.